frontend-react/src/components/MessageTracker/MessageDetails.tsx

```diff
@@ -1,12 +1,16 @@
 import React from "react";
+import moment from "moment";
```

Comment:
Let's try to deprecate our usage of the `moment` dependency -- it adds a huge amount to our payload size, and the maintainers even say that [the project is done](https://momentjs.com/docs/#/-project-status/).

Instead, we can use JavaScript's native `Intl`:

```jsx
// can declare this in the "global" scope outside of components
const dateTimeFormatter = new Intl.DateTimeFormat("en-US", { month: "2-digit", day: "2-digit", year: "numeric", hour: "2-digit", minute: "2-digit", second: "2-digit" })

// call format() whenever you need a formatted date:
content={ dateTimeFormatter.format(messageDetails!.submittedDate) }  
```

Once we've removed all references to `moment` in our codebase, we can add a ticket to remove it from our dependencies and shed a lot of bundle weight :)